migrate_issues.rb

```diff
@@ -165,15 +169,12 @@ def hendl_pr(original)
   end
 end
 
-Hendl.new(source: "fastlane/playground",
-     destination: "fastlane/playground2",
-          reason: "`fastlane` is now a mono repo, you can read more about this decision in our [blog post](https://fastlane.tools). All tools are now available in the [fastlane main repo](https://github.com/fastlane/fastlane).")
-
-# require './tools'
-# names = @tools
-# source = "fastlane/playground" # TODO: Should be fastlane
-# names.each do |current|
-#   Hendl.new(source: "fastlane/#{current}",
-#        destination: "fastlane/fastlane",
-#             reason: "`fastlane` is now a mono repo, you can read more about this decision in our [blog post](https://fastlane.tools). All tools are now available in the [fastlane main repo](https://github.com/fastlane/fastlane).")
-# end
+require './tools'
+names = @tools
+names.delete("fastlane") # we don't want to import issues from our own repo
```

Comment:
`names = @tools.reject { |tool| tool == "fastlane" }` better to copy than mutate in place if something might be shared.